plugin/src/main/java/org/wso2/carbon/uuf/maven/bean/ThemeConfig.java

```diff
@@ -55,15 +64,26 @@ public void setCss(List<String> css) {
      * @return header JS relative paths
      */
     public List<String> getHeadJs() {
+
         return headJs;
     }
 
     /**
      * Sets the head JS relative paths of this theme configuration.
      *
      * @param headJs head JS relative paths to be set
+     * @throws IllegalArgumentException if a header JS file relative path starts with a '/'
      */
     public void setHeadJs(List<String> headJs) {
+        if (headJs != null) {
+            for (String relativePath : headJs) {
+                if (relativePath.charAt(0) == '/') {
+                    throw new IllegalArgumentException(
+                            "Header JS file relative path cannot start with a '/'. '" + relativePath +
```

Comment:
I think this way looks better as the whole message is in a one line, which improves the readability.